src/python/pants/option/global_options.py

```diff
@@ -663,7 +663,7 @@ def register_bootstrap_options(cls, register):
             metavar="<path>",
             # NB: See `--pants-config-files`.
             fingerprint=False,
-            default=["/etc/pantsrc", "~/.pants.rc"],
+            default=["/etc/pantsrc", "~/.pants.rc", ".pantsrc"],
```

Comment:
how about `.pants.rc` to match the file name place in `~` ?

Reply:
I forgot I hate files that use custom extensions for standard format files. So thanks for the reminder :smile:

Reply:
I forgot I hate files that use custom extensions for standard format files. So thanks for the reminder :smile: